setup.py

```diff
@@ -1,11 +1,31 @@
 #!/usr/bin/env python
-#
+import os.path
 
 import setuptools
 
+from sprockets.mixins.amqp import __version__
```

Comment:
I worked around this in the same way as here:

https://github.com/sprockets/sprockets.mixins.correlation/blob/master/sprockets/mixins/correlation/__init__.py